test/e2e/storage/drivers/csi.go

```diff
@@ -373,11 +375,11 @@ var _ testsuites.DynamicPVTestDriver = &gcePDCSIDriver{}
 func InitGcePDCSIDriver() testsuites.TestDriver {
 	return &gcePDCSIDriver{
 		driverInfo: testsuites.DriverInfo{
-			Name:        GCEPDCSIDriverName,
-			FeatureTag:  "[Serial]",
-			MaxFileSize: testpatterns.FileSizeMedium,
+			Name:       GCEPDCSIDriverName,
+			FeatureTag: "[Serial]",
 			SupportedSizeRange: volume.SizeRange{
 				Min: "5Gi",
+				Max: strconv.FormatInt(testpatterns.FileSizeMedium, 10),
```

Comment:
Looks like testpatterns.FileSizeMedium will return type int64 but the size here should be a string like "10Gi", not in KB